glue-codes/openfast-cpp/src/FAST_Prog.cpp

```diff
@@ -103,43 +184,70 @@ int main() {
   iErr = MPI_Comm_size( MPI_COMM_WORLD, &nProcs);
   iErr = MPI_Comm_rank( MPI_COMM_WORLD, &rank);
 
+  int couplingMode ; //CLASSIC (SOWFA style = 0) or STRONG (Conventional Serial Staggered - allow for outer iterations = 1)
   double tEnd ; // This doesn't belong in the FAST - C++ interface 
-  int ntEnd ; // This doesn't belong in the FAST - C++ interface
+  int ntStart, ntEnd ; // This doesn't belong in the FAST - C++ interface
+  bool setExpLawWind; // Set wind speed at Aerodyn nodes based on an exponential profile. Useful for testing the C++ API before running actuator line simulations.
 
   std::string cDriverInputFile="cDriver.i";
   fast::OpenFAST FAST;
   fast::fastInputs fi ;
   try {
-    readInputFile(fi, cDriverInputFile, &tEnd);
+      readInputFile(fi, cDriverInputFile, &tEnd, &couplingMode, &setExpLawWind);
   }
   catch( const std::runtime_error & ex) {
```

Comment:
Why a catch here? You are printing the error, but removing the output of stacktrace that would have otherwise been printed by letting the default handler handle this exception.

Reply:
Agreed. Implemented in 9f34f9eb119d38dc0c78bfe58fdd23d084d02430.